WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -147,7 +147,6 @@ class PageListViewController: AbstractPostListViewController, UIViewControllerRe
 
     override func configureTableView() {
         tableView.accessibilityIdentifier = "PagesTable"
-        tableView.isAccessibilityElement = true
```

Comment:
Why is this removed?

Reply:
If a view is an accessibility element, it makes it so that the elements inside can't be selected within `XCUITest` queries. Tableviews shouldn't be accessibility elements – their cells should be. I tested this change using VoiceOver on-device, and it works the way I'd expect – a user can scroll through each cell individually.